src/compiler/parse/state/tag.ts

```diff
@@ -378,6 +378,13 @@ function read_attribute(parser: Parser, unique_names: Set<string>) {
 	if (type) {
 		const [directive_name, ...modifiers] = name.slice(colon_index + 1).split('|');
 
+		if (directive_name === '') {
+			parser.error({
+				code: 'empty-directive-name',
+				message: `${type} name cannot be empty`
+			}, start+colon_index);
```

Comment:
the error location seems to be off by 1, looking at the test cases, the error message is pointing at 1 character in front of the colon.

Reply:
the error location is pointing to the colon character because if we write `<element use:/>`
it will point to the / character.

I'm not sure if it does matter but I think the current one is more clear.

Reply:
I updated the error offset as you suggested 👍

Reply:
I made the changes. Could you check it please?